hal/include/HAL/CAN.h

```diff
@@ -11,6 +11,7 @@
 
 #include "FRC_NetworkCommunication/CANSessionMux.h"
 
+extern "C" {
 void canTxSend(uint32_t arbID, uint8_t length,
                int32_t period = CAN_SEND_PERIOD_NO_REPEAT);
```

Comment:
Putting extern C around a function with a default parameter... I don't think this works, as I think it will end up as a C++ function anyway (only reasonable outcome of doing this I can think of)?